doc/content/gateways/mikrotik-knot/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "MikroTik KNOT"
+description: ""
+weight: 1
+---
+
+{{< figure src="mikrotik-knot.jpeg" alt="Ltmikrotik-knotAP" class="plain float" >}}
```

Comment:
You should move this image, as it gets covered by the warning about changing frequency plans

Reply:
<img width="1391" alt="Bildschirmfoto 2021-05-03 um 13 50 09" src="https://user-images.githubusercontent.com/6963436/116868407-68f45380-ac17-11eb-9ac3-012c66b2e83e.png">